File: microservices/usuarios-service/src/main/java/cl/duoc/levelup/usuarios_service/model/Usuario.java
package cl.duoc.levelup.usuarios_service.model;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Table(name = "usuarios")
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Usuario {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String nombre;

    @Column(unique = true)
    private String email;

    private Integer edad;

    private String password;

    private String preferencias;

    private String direccion;

    private String rol;

}

File: mobile-app/app/src/main/java/com/levelup/data/model/Usuario.kt
package com.levelup.data.model

data class Usuario(
    val id: Long,
    val nombre: String,
    val email: String,
    val edad: Int? = null,
    val preferencias: String? = null,
    val direccion: String? = null,
    val rol: String? = null,
    val password: String
)